src/components/TodosFilter.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useCallback, useContext, useMemo } from 'react';
+import classNames from 'classnames';
+import { TodosContext } from '../contexts/TodosContext';
+import { TodoActions } from '../types/TodoActions';
+
+export const TodosFilter: React.FC = () => {
+  const { selectedFilter, setSelectedFilter } = useContext(TodosContext);
+
+  const TodoActionsToArr = useCallback((data: typeof TodoActions): string[] => {
```

Comment:
```suggestion
  const todoActionsToArr = useCallback((data: typeof TodoActions): string[] => {
```
camelCase for function names
but you don't need it at all - just use `Object.values(TodoActions)` w/o memorization (this process also need some resources etc.) or at least return  `Object.values(TodoActions)` from useMemo callback